test/e2etest/test-config.tfvars.tmpl

```diff
@@ -64,6 +64,9 @@ number_of_bamboo_agents = 3
 # for details
 bamboo_dataset_url = "https://bamboo-test-datasets.s3.amazonaws.com/testing_dataset_minimal.zip"
 
+# To fix "pvc still exists with finalizers" error while uninstalling
```

Comment:
I suggest to rephrase this comment. Something like `to force shutdown bamboo immediately`
But up to you.